src/TableOfContents/index.tsx

```diff
@@ -382,12 +395,24 @@ const TableOfContentsItemInner = ({
 /**
  * Traverses contents backwards to find the first index with a lower depth
  */
-function findParentIndex(currentDepth: number, contents: TableOfContentsItem[]) {
+function findParentIndex(currentDepth: number, contents: TableOfContentsItem[]): number {
   for (let index = contents.length - 1; index >= 0; index--) {
     if (contents[index].depth === currentDepth - 1) {
-      return String(index);
+      return index;
```

Comment:
Why was it string before? Kind of weird.

Reply:
Yes, dunno.